AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -112,6 +116,7 @@ - (void)tearDown {
   [self waitForExpectations:@[ expection ] timeout:1];
 
   // Clear
+  [OHHTTPStubs removeAllStubs];
```

Comment:
Just nit-picking. If we have `MSHttpTestUtil`, why not move this line over there, too? I know that it is just one line, but it seems to be more concise to me.